### ISSUE TITLE
Post-build event fails for PDB

### ISSUE THREAD

**Issue by cocowalla** [NationalSecurityAgency/ghidra#30]
**Describe the bug**
The post-build event for the PDB project fails, because of:

1. Extra double quotes
2. A trailing slash in the output path (it's possible this works on some versions of xcopy, but not with mine)

**To Reproduce**
1. Build the PDB project in `Ghidra\Features\PDB\src\pdb`

**Expected behavior**
The post-build event should succeed, copying output files into `Ghidra\Features\PDB\build\os\win64`

**Environment (please complete the following information):**
 - OS: Windows 7 SP1

**Additional context**
This can be resolved by updating the post-build event to:

```
copy /y /v /b "$(VSInstallDir)DIA SDK\bin\amd64\msdia*.dll" "$(OutDir)"
set OS_DIR=$(SolutionDir)..\..\os
if exist "%OS_DIR%\win64" (
   xcopy /Y /S "$(OutDir.TrimEnd('\'))" "%OS_DIR%\win64"
)
```

**Comment by jonyluke** [comment]
So what do you have to change here? @cocowalla 
```
<Target Name="PostBuildEvent" Condition="'$(PostBuildEventUseInBuild)'!='false'">
    <Message Text="%(PostBuildEvent.Message)" Condition="'%(PostBuildEvent.Message)' != '' and '%(PostBuildEvent.Command)' != ''" Importance="High" />
    <Exec Command="%(PostBuildEvent.Command)$(_BuildSuffix)" Condition="'%(PostBuildEvent.Command)' != ''"/>
  </Target>
```

**Comment by cocowalla** [comment]
@jonyluke not sure where you got that, but it doesn't appear to be in `pdb.vcxproj`. You want to change the `PostBuildEvent` elements in `pdb.vcxproj` (note there is one for debug builds, and another for release builds):

```xml
<PostBuildEvent>
  <Command>
copy /y /v /b "$(VSInstallDir)DIA SDK\bin\amd64\msdia*.dll" "$(OutDir)"
set OS_DIR=$(SolutionDir)..\..\os
if exist "%OS_DIR%\win64" (
   xcopy /Y /S "$(OutDir.TrimEnd('\'))" "%OS_DIR%\win64"
)
  </Command>
</PostBuildEvent>
```

**Comment by jonyluke** [comment]
@cocowalla
I am still getting an error when trying to load a pdb 


```
PDB Execution failure of pdb.exe.
This was likely caused by severe execution failure which can occur if executed
on an unsupported platform. It may be neccessary to rebuild the PDB executable
for your platform (see Ghidra/Features/PDB/src).
ghidra.app.util.bin.format.pdb.PdbException: PDB Execution failure of pdb.exe.
This was likely caused by severe execution failure which can occur if executed
on an unsupported platform. It may be neccessary to rebuild the PDB executable
for your platform (see Ghidra/Features/PDB/src).
	at ghidra.app.util.bin.format.pdb.PdbParserNEW.verifyPdbSignature(PdbParserNEW.java:568)
	at ghidra.app.util.bin.format.pdb.PdbParserNEW.processPdbContents(PdbParserNEW.java:549)
	at ghidra.app.util.bin.format.pdb.PdbParserNEW.parse(PdbParserNEW.java:149)
	at pdb.LoadPdbTask$1.analysisWorkerCallback(LoadPdbTask.java:59)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager$AnalysisWorkerCommand.applyTo(AutoAnalysisManager.java:1683)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager$AnalysisTaskWrapper.run(AutoAnalysisManager.java:685)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:785)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:664)
	at ghidra.app.plugin.core.analysis.AutoAnalysisManager.startAnalysis(AutoAnalysisManager.java:629)
	at ghidra.app.plugin.core.analysis.AnalysisBackgroundCommand.applyTo(AnalysisBackgroundCommand.java:62)
	at ghidra.framework.plugintool.mgr.BackgroundCommandTask.run(BackgroundCommandTask.java:101)
	at ghidra.framework.plugintool.mgr.ToolTaskManager.run(ToolTaskManager.java:315)
	at java.base/java.lang.Thread.run(Thread.java:834)

```

**Comment by cocowalla** [comment]
@jonyluke yep, me too :)

But I think that's a separate issue, this one just describes an issue with the post-build event.

**Comment by ghidra1** [comment]
> Solution in #94
> 
> @cocowalla
> I am still getting an error when trying to load a pdb
> 
> ```
> PDB Execution failure of pdb.exe.
> This was likely caused by severe ...
>```

The above is likely cause by not having registered the msdia140.dll (see docs/README_PDB.html).
Detection and reporting of this condition has been improved for 9.0.2.

**Comment by ghidra1** [comment]
The post-build event quote issue has been resolved for 9.0.2.
